spinta/datasets/backends/helpers.py

```diff
@@ -24,3 +27,29 @@ def handle_ref_key_assignment(keymap: KeyMap, value: Any, prop: Property) -> dic
             val = keymap.encode(prop.dtype.model.model_type(), value)
             val = {'_id': val}
     return val
+
+
+def detect_backend_from_content_type(context, content_type):
+    config = context.get('config')
+    backends = config.components['backends']
+    for backend in backends.values():
+        if issubclass(backend, ExternalBackend) and backend != BackendNotImplemented:
```

Comment:
```suggestion
        if issubclass(backend, ExternalBackend) and not issubclass(backend, BackendNotImplemented):
```

Always use `issubclass` or `isinstance`, when comparing classes, to take into account class inheritance.